Skully (#3797)
freesewing/freesewing: markdown/org/docs/patterns/skully/instructions/en.md

---
title: "Skully (the logo): Sewing Instructions"
---

<Tip>

A lot of the smaller seams (eyes, nose) are best sewn by hand. 

Most of the seams will benefit from being basted before being sewn.

</Tip>

## Step 1: The darts

There are three darts that need to be closed first. Pin and sew them, then remove the bulk and
give them a good press. Repeat for the other three darts.

## Step 2: The eyes

Sew together the sides of the eye, so you will make a cone. Cut away any bulk. Repeat for the other eye

## Step 3: The nose

> Like the eyes, the nose too has parts that need to be sewn together. There are two of these sides that need to be sew together. This creates two connected cones. This is very finicky, so take your time.

Reviewer: I am bit confused by this bit. Is it the straight edges that are sewn together or the curved edges of the nose?

Author: The straight edges. There is a plug-in that shows how, but it is stalled in the plugin-annotations commit.

Reviewer: ```suggestion
Like the eyes, the nose has seams that need to be sewn together. Sew the two straight seams together creating two connected cones. This is very finicky, so take your time.
```
Would this be better than.

it partly. 

</Tip>

When you are sewing together the individual parts, you will be making two halves of the skull. 

Sewing the eyes and nose to the cheek and forehead parts is best done by hand. You will have to aling
the different notches to their corresponding ones on the other part. 

Sewing the lowerJaw to the lowerMouth and upperMouth is challenging. You will be making a normal seam 
between the lowerJaw parts and the lowerMouth part. You will then have to continue this seam along
the dashed line (14) on the upperMouth part. Baste or lots of pins are your friend. And make sure you 
first visualize how this will get together before doing the final sewing.

The back of the upperMouth is not sewn to anything. It is just there to provide stability.

## Step 5: Hair

Now is the time to add hair to skully. When putting the embroidery floss in, make sure you add knots 
on the inside, or else it is very easy to pull Skully's hair out later. 

## Step 6: Stuffing

- Fill the plush toy with stuffing through the opening you left, making sure to fill all the parts 
well. The areas in the front are the hardest to get to.

> - _Slipsticth_ or _whipstitch_ the opening closed.

Reviewer: Ok I will suggest one that has the bullet points once the other comment is resolved.

---
title: "Skully (the logo): Sewing Instructions"
---

<Tip>

A lot of the smaller seams (eyes, nose) are best sewn by hand. 

Most of the seams will benefit from being basted before being sewn.

</Tip>

## Step 1: The darts

> There are three darts that need to be closed first. Pin and sew them, then remove the extra fabric and
> give them a good press. Repeat for the other three darts.
> 
> ## Step 2: The eyes
> 
> Sew together the sides of the eye, so you will make a cone. Cut away any excess seam allowance. Repeat for the other eye
> 
> ## Step 3: The nose
> 
> Like the eyes, the nose has seams that need to be sewn together. Sew the two straight seams together creating two connected cones. This is very finicky, so take your time.
> 
> ## Step 4: The head
> 
> <Tip>
> 
> The Skully pattern is set up to be sewn-by-numbers. Every seam is numbered. Please start at '1', and
> keep going until you reach '21'. This is the seam through which you will fill Skully, so only close
> it partly. 
> 
> </Tip>
> 
> When you are sewing together the individual parts, you will be making two halves of the skull. 
> 
> Sewing the eyes and nose to the cheek and forehead parts is best done by hand. You will have to align
> the different notches to their corresponding ones on the other part. 
> 
> Sewing the lowerJaw to the lowerMouth and upperMouth is challenging. You will be making a normal seam 
> between the lowerJaw parts and the lowerMouth part. You will then have to continue this seam along
> the dashed line (14) on the upperMouth part. Baste or lots of pins are your friend. And make sure you 
> first visualize how this will get together before doing the final sewing.
> 
> The back of the upperMouth is not sewn to anything. It is just there to provide stability.
> 
> ## Step 5: Hair
> 
> Now is the time to add hair to skully. When putting the embroidery floss in, make sure you add knots 
> on the inside, or else it is very easy to pull Skully's hair out later. I threaded two pieces of floss
> through a needle, and stick it from the outside through the head. Then make an overhand knot, and push
> the needle back out. You should make shorter hair towards the outside, and longer in the middle.
> 
> ## Step 6: Stuffing
> 
> - Fill the plush toy with stuffing through the opening you left, making sure to fill all the parts 
> well. The areas in the front are the hardest to get to.
> - _Slipsticth_ or _whipstitch_ the opening closed.
> 
> ## Step 6: Tweaks
> 
> You can use the long upholstery needle to add some tweaks to Skully. Adding some stitches, attaching 
> the back of the eyes to corner of seams 14 and 21 right through the skull helps with keeping the 
> eyes from popping out, and can help with giving the head more form. The same applies to the nose.
> 
> It is also a good isea to add stitches between seams 5 and 14, along the length of both seams. This
> will make the cheek bones more pronounced and adds to the overall shape of the skull.
> 
> ## Step 7: Enjoy!
> 
> That's it, you are all done. Start planning what pirate sewing ship you'll be joining!

Reviewer: ```suggestion
There are three darts that need to be closed first. 

- Pin and sew them, then remove the extra fabric and give them a good press. Repeat for the other three darts.

## Step 2: The eyes

-  Sew together the sides of the eye, so you will make a cone. Cut away any excess seam allowance. Repeat for the other eye

## Step 3: The nose

Like the eyes, the nose has seams that need to be sewn together. 

- Sew the two straight seams together creating two connected cones. This is very finicky, so take your time.

## Step 4: The head

<Tip>

The Skully pattern is set up to be sewn-by-numbers. Every seam is numbered. Please start at '1', and keep going until you reach '21'. This is the seam through which you will fill Skully, so only close it partly. 

</Tip>

When you are sewing together the individual parts, you will be making two halves of the skull. 

Sewing the eyes and nose to the cheek and forehead parts is best done by hand. You will have to align the different notches to their corresponding ones on the other part. 

Sewing the lowerJaw to the lowerMouth and upperMouth is challenging. You will be making a normal seam between the lowerJaw parts and the lowerMouth part. You will then have to continue this seam along the dashed line (14) on the upperMouth part. Baste or lots of pins are your friend. And make sure you first visualize how this will get together before doing the final sewing.

The back of the upperMouth is not sewn to anything. It is just there to provide stability.

## Step 5: Hair

Now is the time to add hair to skully. When putting the embroidery floss in, make sure you add knots on the inside, or else it is very easy to pull Skully's hair out later. I threaded two pieces of floss through a needle, and stick it from the outside through the head. Then make an overhand knot, and push the needle back out. You should make shorter hair towards the outside, and longer in the middle.

## Step 6: Stuffing

- Fill the plush toy with stuffing through the opening you left, making sure to fill all the parts well. The areas in the front are the hardest to get to.
- _Slipsticth_ or _whipstitch_ the opening closed.

## Step 6: Tweaks

You can use the long upholstery needle to add some tweaks to Skully. Adding some stitches, attaching the back of the eyes to corner of seams 14 and 21 right through the skull helps with keeping the eyes from popping out, and can help with giving the head more form. The same applies to the nose.

It is also a good idea to add stitches between seams 5 and 14, along the length of both seams. This will make the cheek bones more pronounced and adds to the overall shape of the skull.

## Step 7: Enjoy!

That's it, you are all done. Start planning what pirate sewing ship you'll be joining!
```
